build.gradle

```diff
@@ -25,7 +25,8 @@ dependencies {
     compile 'com.google.code.gson:gson:2.8.0'
     compile 'com.google.guava:guava:21.0'
     testCompile 'junit:junit:4.12'
-    testCompile 'org.mockito:mockito-core:2.7.22'
+    testCompile group: 'org.powermock', name: 'powermock-module-junit4', version: '1.6.6'
```

Comment:
Why do we need powermock?

Reply:
It provides capabilities to overcome the problems of Mockito, such as the lack of ability to mock final, static or private methods.

Reply:
I made some experiments in the HostDriverFixture and we jsut discussed to improve the tests in a next ticket ....1141. So we can test methods like "initializeStartAndEndForRowAndColumn(elementLocator)" in the LocatorByStartStop Class perfectly. :+1: